all/entrypoint.sh

```diff
@@ -458,7 +462,7 @@ if [ $GEN_GATEWAY = true ]; then
     mkdir -p ${GATEWAY_DIR}
 
     protoc $PROTO_INCLUDE \
-		--grpc-gateway_out=logtostderr=true:$GATEWAY_DIR ${PROTO_FILES[@]}
+		--grpc-gateway_out=logtostderr=true${GATEWAY_OPTS}:$GATEWAY_DIR ${PROTO_FILES[@]}
```

Comment:
Are there other options you are interested in atm other than `source_relative`?

I ask because this tool, for the most part, manage options internally and expose them via specific flags.
For example, it doesn't support adding options on the `--go_out` protoc arg, and instead different flags like `--go-source-relative` are responsible for that.
So for `source_relative`, I would suggest setting that option on `grpc-gateway_out` if `--go-source-relative` is used (There is actually a competing PR for that)

This is not to say there isn't value supporting other options, so please let me know and we can think about it then.

Reply:
For now, I only need `source_relative`. But perhaps this would be helpful for other options. I just noticed https://github.com/namely/docker-protoc/pull/248. Seems like a more consistent approach if only `source_relative` is needed. Feel free to close this PR if https://github.com/namely/docker-protoc/pull/248 is more preferable. I'm using my fork right now for this particular feature and will switch once either this PR or #248 is merged

Reply:
Thank you, I think we should go with #248 approach for now though we're missing some tests. 
If you want to contribute to that PR it would be greatly appreciated :)